### PR TITLE
Changed gulp file and file structure

### PR DESCRIPTION
**Pull request by Taylord93**

### CODE REVIEW

**Comment by Taylord93** [comment]
Just made some adjustments for issues i found while starting up a new projects. Should be cleaner.

File: gulpfile.js [hunk 1]
```diff
@@ -143,17 +134,10 @@ gulp.task('tests', function(done) {
...
-
 // task chains
-gulp.task('default', ['styles', 'fonts', 'views', 'css-vendors', 'js-vendors', 'scripts', 'watch']); // build app
+gulp.task('default', ['styles', 'fonts', 'views', 'js-vendors', 'scripts', 'watch']); // build app
 gulp.task('serve', ['styles', 'scripts', 'fonts', 'views', 'browser-sync-server', 'watch']); // launch server
 gulp.task('build', ['styles', 'scripts', 'fonts', 'views', 'browser-sync', 'watch']); // proxy
-gulp.task('vendors', ['css-vendors', 'js-vendors']); // vendor js and css
+gulp.task('vendors', ['js-vendors']); // vendor js and css
 gulp.task('test', ['tests', 'watch']);
 gulp.task('test-once', ['tests']);
```
Comment (Toddses) [inline gulpfile.js]: We might want to drop all these changes. I made a lot of changes to the gulpfile in #12. For the most part I've covered your changes here, plus many more. At the very least, it's going to cause merge conflicts.

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
   "dependencies": {
     "angular": "^1.5.7",
     "angular-resource": "^1.5.7",
-    "angular-ui-router": "^0.3.1"
+    "angular-ui-router": "^0.3.1",
+    "normalize-sass": "^1.0.0"
```
Comment (Toddses) [inline package.json]: In #12 I pulled in `normalize.css`. I see no reason to use a sass port when the original can compile into the sass just as easily.

File: src/scss/app.scss [hunk 3]
```diff
@@ -1,10 +1,9 @@
 @import '../../node_modules/bourbon/app/assets/stylesheets/bourbon';
 @import '../../node_modules/bourbon-neat/app/assets/stylesheets/neat';
+@import '../../node_modules/normalize-sass/normalize';
```
Comment (Toddses) [inline src/scss/app.scss]: See my comment above regarding using `normalize.css` over the sass port.

**Comment by Toddses** [comment]
@Taylord93 Be sure to do a review of PR #12 before we consider merging this. Some of these changes are already in that PR and some others will cause merge conflicts. We should consider #12 and merge ASAP as it will have an affect on most of the build process.

**Comment by Taylord93** [comment]
Noted. I was just looking through a lot of the other changes. I only put in a PR for this as it was local from a while ago that I never pushed up.

**Comment by Taylord93** [comment]
Issues solved here resolved in #12

**Pull request closed without merging**